load-gen/src/main/java/io/smartcat/data/loader/api/Terminateable.java

```diff
@@ -0,0 +1,13 @@
+package io.smartcat.data.loader.api;
+
+/**
+ * Has ability to terminate itself.
+ */
+public interface Terminateable {
```

Comment:
I'm wondering. If I introduce AutoCloseable, is than still responsibility of LoadGenerator to close DataSource, Worker and RateGenerator or this responsibility should be outside?